src/app/projectsHome/page.tsx

```diff
@@ -0,0 +1,8 @@
+import ProjectPic from "@/public/images/Projectshome.svg";
+import Image from "next/image";
+
+const ProjectsHome = () => {
+  return <Image alt="header" src={ProjectPic} />;
```

Comment:
instead of having this within a page.tsx file, create a component within the components/projects and call it ProjectsHeader.tsx and make it so that the image is h-screen w-screen and then absolutely position the projects text.

Reply:
For some reason I cant save the picture without the text "Projects". Is there any way I can remove the text from the figma, save the picture, then I'll put it back. It wont let me edit the figma to do that.

Reply:
check my comment on discord!

Reply:
ok fixed it! also in my issue the picture of the header is from the dark mode version i js did the light mode version from the figma tho bc i remember someone saying that in the meeting to not worry about dark mode rn

Reply:
OHH ALSOO should i change my text-9xl to the vh vw thing u were saying lowkey forgot abt that